web/src/compose_send_menu_popover.ts

```diff
@@ -16,25 +17,27 @@ import * as popover_menus from "./popover_menus.ts";
 import * as scheduled_messages from "./scheduled_messages.ts";
 import {parse_html} from "./ui_util.ts";
 import {user_settings} from "./user_settings.ts";
+import * as util from "./util.ts";
 
 export const SCHEDULING_MODAL_UPDATE_INTERVAL_IN_MILLISECONDS = 60 * 1000;
 const ENTER_SENDS_SELECTION_DELAY = 600;
 
 let send_later_popover_keyboard_toggle = false;
 
-function set_compose_box_schedule(element) {
-    const selected_send_at_time = element.dataset.sendStamp / 1000;
+function set_compose_box_schedule(element: HTMLElement): number {
+    assert(element.dataset.sendStamp !== undefined);
+    const selected_send_at_time = Number.parseInt(element.dataset.sendStamp, 10) / 1000;
     return selected_send_at_time;
 }
 
 export function open_schedule_message_menu(
-    remind_message_id = undefined,
-    target = "#send_later i",
-) {
+    remind_message_id: number | undefined = undefined,
+    target: tippy.ReferenceElement = util.the($("#send_later i")),
```

Comment:
Why switch `target` default from a string to a jQuery element here? 

I think the right cleanup is probably for `target` to be computed in the two callers that don't pass it, rather than having this default.